growth/too/templates/event_header.html

```diff
@@ -1,5 +1,5 @@
 <div class="d-flex justify-content-between">
-    <h5 {% if event.retracted %}class="retracted"{% endif %}>
+    <h5 class="{% for tag in event.tags %}{{tag}} {% endfor %}">
```

Comment:
Please use your prefixed CSS class names consistently here.

Reply:
Leo, the issue here is that I think if I prefix them all the same, the background color associated with the different classifications will also be applied to the date and time rather than just to the pillboxes. But I've added the 'tag' prefixes to the retracted and mdc classes, while the other tags are prefixed by 'cls'.

Reply:
Right, but using the same class names, you can single out which elements you want to refer to using the following CSS selectors:

```css
h5.tag-retracted {
   ...
}

.badge.tag-retracted {
   ...
}
```